packages/block-library/src/post-comment/edit.js

```diff
@@ -34,8 +34,7 @@ export default function Edit( { attributes: { commentId }, setAttributes } ) {
 					) }
 				>
 					<TextControl
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
This is a deprecated block so you need to modify the `block.json` to make it appear. (Delete the `__experimental` field, and change `inserter` to `true`.)

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/006f4a1e-ca6e-43de-8cdc-973b9882dfd9" alt="Post Comment block placeholder, before" width="651">|<img src="https://github.com/user-attachments/assets/99af3c9c-67bf-468b-84ff-f576fea08721" alt="Post Comment block placeholder, after" width="651">|